app/src/main/java/com/nutomic/syncthingandroid/service/SyncthingService.java

```diff
@@ -686,13 +690,17 @@ public boolean importConfig() {
         File config = new File(Constants.EXPORT_PATH, Constants.CONFIG_FILE);
         File privateKey = new File(Constants.EXPORT_PATH, Constants.PRIVATE_KEY_FILE);
         File publicKey = new File(Constants.EXPORT_PATH, Constants.PUBLIC_KEY_FILE);
+        File httpsCert = new File(Constants.EXPORT_PATH, Constants.HTTPS_CERT_FILE);
+        File httpsKey = new File(Constants.EXPORT_PATH, Constants.HTTPS_KEY_FILE);
         if (!config.exists() || !privateKey.exists() || !publicKey.exists())
             return false;
         shutdown(State.INIT, () -> {
             try {
                 Files.copy(config, Constants.getConfigFile(this));
                 Files.copy(privateKey, Constants.getPrivateKeyFile(this));
                 Files.copy(publicKey, Constants.getPublicKeyFile(this));
+                Files.copy(httpsCert, Constants.getHttpsCertFile(this));
+                Files.copy(httpsKey, Constants.getHttpsKeyFile(this));
```

Comment:
You also need to handle the exception separately and log a different message, given https files failing/not being present isn't a failure. Ideally we'd first check for presence, and then only try to copy if present (in which case the failure log line is warranted).